app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= button_to( "Log Out",destroy_user_session_path, method: :delete) %> 
+
+  <div class="expense-card">
+  
+
+   <div class='expense-heading'>
+    <h1>EXPENSES</h1>
+   </div>
+   
+    <% @expenses.each do |expense| %>
+      <%= render expense %>
+    <% end %>
+
+   <%= link_to "Add New expense", new_expense_path, class:'expenses-footer' %> 
```

Comment:
- [ ] Please 🙏🏽 use uppercases to be similar to the original design.

Reply:
Thank you. This has been implemented